_includes/navbar.html

```diff
@@ -31,6 +31,12 @@
                 <li><a href="/custom-module-development/" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="custom-module-dev">Custom Module Development</a></li>
               </ul>
             </li>
+            <li class="dropdown js-nav-item-central">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Use Cases <i class="fa fa-caret-down" aria-hidden="true"></i></a>
+              <ul class="dropdown-menu dropdown-menu-left" role="menu">
+                <li><a href="/guides/" ga-on="click" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="guides">Development Guides</a></li>
```

Comment:
The use cases should list the individual deployment guides, followed by a "see all":

1. Configure a production-grade AWS account structure
1. Configure a production-grade VPC on AWS
1. Deploy a production-grade Kubernetes cluster on AWS
1. See all production deployment guides -->

Reply:
Ooh I see